swaggerpy/response.py

```diff
@@ -53,7 +52,7 @@ def cancel(self):
         """Try to cancel the API (meaningful for Asynchronous client)
         """
         self._cancelled = True
-        self._http_client.cancel(self._request)
+        self._request.cancel()
```

Comment:
for synchronous, `SynchronousEventual` should have also defined `cancel()` for the above call?